.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
```

Comment:
nit: I think this is not related to this project, rather related to developers environment, so devs should ignore in their per-user gitignore?